doc/manual/oqum/hazard/01d_config_classical.tex

```diff
@@ -146,19 +146,32 @@
 
 \begin{minted}[firstline=1,linenos=true,firstnumber=23,fontsize=\footnotesize,frame=single,bgcolor=lightgray]{ini}
 [site_params]
-site_model_file = site_model.xml
+site_model_file = site_model.csv
 \end{minted}
 
 The file containing the site model has the following structure:
 
 \begin{listing}[htbp]
-  \inputminted[firstline=1,firstnumber=1,fontsize=\footnotesize,frame=single,linenos,bgcolor=lightgray]{xml}{oqum/hazard/verbatim/input_site_model.xml}
+  \inputminted[firstline=1,firstnumber=1,fontsize=\footnotesize,frame=single,linenos,bgcolor=lightgray]{xml}{oqum/hazard/verbatim/input_site_model.csv}
   \caption{Example site model input file}
   \label{lst:input_site_model}
 \end{listing}
 
+Notice that the $0$ for the field \texttt{vs30measured} means that the
+\texttt{vs30} field is inferred, not measured. This is the default, so you
+can just skip the field. Also, most of the GMPEs are not sensitive to
```

Comment:
Suggest removing "Also, most of the GMPEs are not sensitive to it."

Reply:
Done